mmv1/products/securitycenterv2/FolderMuteConfig.yaml

```diff
@@ -0,0 +1,111 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'FolderMuteConfig'
+base_url: folders/{{folder}}/locations/{{location}}/muteConfigs
+self_link: 'folders/{{folder}}/locations/{{location}}/muteConfigs/{{mute_config_id}}'
+create_url: folders/{{folder}}/locations/{{location}}/muteConfigs?muteConfigId={{mute_config_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'folders/{{folder}}/locations/{{location}}/muteConfigs/{{mute_config_id}}'
+description: |
+  Mute Findings is a volume management feature in Security Command Center
+  that lets you manually or programmatically hide irrelevant findings,
+  and create filters to automatically silence existing and future
+  findings based on criteria you specify.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/folders.muteConfigs'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_folder_mute_config_basic'
+    primary_resource_id: 'default'
+    external_providers: ["random", "time"]
+    skip_test: true
+    vars:
+      mute_config_id: 'my-config'
+      folder_display_name: "folder-name"
+    test_env_vars:
+      org_id: :ORG_ID
+    test_vars_overrides:
+      sleep: "true"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: folder
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The folder whose Cloud Security Command Center the Mute
+      Config lives in.
+  - !ruby/object:Api::Type::String
+    name: location
+    immutable: true
+    url_param_only: true
+    default_value: global
+    description: |
+      location Id is provided by folder. If not provided, Use global as default.
+  - !ruby/object:Api::Type::String
+    name: mute_config_id
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Unique identifier provided by the client within the parent scope.
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    output: true
```

Comment:
Is this field always an output field?

Reply:
It's not mentioned in the document. I am removing it.

Reply:
Its not mentioned in the doccument. I removed it.